src/python_testing/TC_MCORE_FS_1_3.py

```diff
@@ -68,45 +68,32 @@ def teardown_class(self):
         super().teardown_class()
 
     async def create_device_and_commission_to_th_fabric(self, kvs, port, node_id_for_th, device_info):
-        # TODO: confirm whether we can open processes like this on the TH
         app = self.user_params.get("th_server_app_path", None)
```

Comment:
Would you mind also doing this to all the other test scripts that use th_server_app_path

Reply:
Done.